extensions/EXT/EXT_texture_mirror_clamp_to_edge.txt

```diff
@@ -0,0 +1,151 @@
+Name
+
+    EXT_texture_mirror_clamp_to_edge
+
+Name Strings
+
+    GL_EXT_texture_mirror_clamp_to_edge
+
+Contact
+
+    Christophe Riccio, Unity Technologies (christophe.riccio 'at' unity3d.com)
+
+Contributors
+
+    Contributors to ARB_texture_mirror_clamp_to_edge
+
+Notice
+
+    Copyright (c) 2017 The Khronos Group Inc. Copyright terms at
+        http://www.khronos.org/registry/speccopyright.html
+
+Status
+
+    Complete on September 4, 2017.
+
+Version
+
+    Last Modified Date: September 7, 2017
+    Revision 2
+
+Number
+
+    ES Extension #291
+
+Dependencies
+
+    OpenGL ES 2.0 is required.
+
+    This extension is written against the OpenGL ES 3.2 Specification.
+
```

Comment:
This extension interacts with OES_texture_3D. 

This extension interacts with OpenGL ES 3.0.

This extension interacts with OpenGL ES 3.2.